### ISSUE TITLE
Exception in the explain inference function of DL Query

### ISSUE THREAD

**Issue by ummAssiya** [protegeproject/protege#298]
In Protege 5.0.0 Beta Build 21, the "explain inference" function of the DL Query Tab is not working. An exception is thrown when "explain inference" is called (See attached log). 
[DLQueryExplainInference.txt](https://github.com/protegeproject/protege/files/68280/DLQueryExplainInference.txt)

**Comment by matthewhorridge** [comment]
This works for me.  Can you provide an ontology (zipped up) and the particular result that you want an explanation for?

**Comment by ummAssiya** [comment]
The ontology was created for test purposes, it has only three classes and one individual (see attached file). 
The query in DL Query Tab is very simple: It asks for all individuals classified as "Person".  I would expect the the explanation to say something like: "Stewie" asserted a Baby, Baby subclass of Child, Child subclass of Person. 
I used the button with the question mark to call "explain inference". See screenshot
![explaininference](https://cloud.githubusercontent.com/assets/6398283/11940859/9d32c29e-a82b-11e5-91fb-c6593cf47588.jpg)
[StewieTheBaby.zip](https://github.com/protegeproject/protege/files/68962/StewieTheBaby.zip)

**Comment by tudorache** [comment]
@ummAssiya Can you please try to clear the Protege cache from your home folder -> .Protege. Just delete the .Protege folder and try again.

I did not replicate the exception, so you may have another version of the explanation plugin running. 

If you still have this issue, please post the entire output from the console.

**Comment by ummAssiya** [comment]
@tudorache Thank you. As advised, I deletet the cache folder. Protege 5.0 was then running without plugins. After that, in the Protege menu I used  " File --> Check for plugins ..." in order to install the plugins. I installed 
Pellet Reasoner Plug-in (2.2.0)
DL Query (3.0.0)
Factplusplus Plug-in (1.6.4). 

The described problem with the "explain inference" functionality in the DL Query Tab still exists. ( I tried the combination of DL Query and Pellet as reasoner, and also DL Query and Fact++, both create the same exception when "explain reference" is called).
I am including both the protege.log (in the protege.zip file) and the console output.
[ProtegeConsole.txt](https://github.com/protegeproject/protege/files/69413/ProtegeConsole.txt)


[protege.zip](https://github.com/protegeproject/protege/files/69412/protege.zip)

**Comment by matthewhorridge** [comment]
This isn't reproducible for me using beta-21 either.  @ummAssiya It looks like you might have some old version of the explanation plugin lying around somewhere.  Please can you create a new installation of Protege (in a different directory from your current installation) and try to repeat this?

**Comment by ummAssiya** [comment]
I deleted the .Protege folder and the Protege-related files I could find in my temp folder. I have two older installations of Protege and renamed their folders temporarily. Then I unpacked the protege beta-21 download anew.  The error would still persist.
Then, I dowloaded the beta-21 version again from the protege website. Unpacked it. Tested, now it works. 
Strangely, there seems to be some differences in the two protege versions, which both are named 5.0.0 build 21. The first one (I downloaded it on 21st of December, 2015) and the second one (downloaded today, from here http://protege.stanford.edu/products.php#desktop-protege) seem not to be identical. 
In any case, now it works, thank you.
